lib/commands/test-init.ts

```diff
@@ -37,6 +37,25 @@ class TestInitCommand implements ICommand {
 				'save-dev': true,
 				optional: false,
 			});
+
+			let modulePath = path.join(projectDir, "node_modules", mod);
+			let modulePackageJsonPath = path.join(modulePath, "package.json");
+			let modulePackageJsonContent = this.$fs.readJson(modulePackageJsonPath);
+			let modulePeerDependencies = modulePackageJsonContent.peerDependencies || {};
+
+			for (let peerDependency in modulePeerDependencies) {
+				let dependencyVersion = modulePeerDependencies[peerDependency] || "*";
+
+				// catch errors when a peerDependency is already installed
+				// e.g karma is installed; karma-jasmine depends on karma and will try to install it again
```

Comment:
As the comment says, in case the dependency `karma` is already installed, the current implementation will reinstall it, but it may also change its version, which I do not think is expected. For example in case I have installed `karma`: `1.6.0`, but the karma-jasmine has peerDependency: `"karma":"~0.9"`, I'll end-up with the version `~0.9`. Can we skip the installation of dependencies in case we already have the same dependency installed.